### DISCUSSION TITLE
How to make the dark part totally black?

### DISCUSSION THREAD

**Discussion by OriLiMu** [darkreader/darkreader#8016] [Q&A] [answered]
I want the dark part of web page totally black if the current color hex code (after dark mode process) is lower than #111111(for example), the final hex code will become #000000, I try to modify the css file after click the `dev` button, but so complicated for me. So I ask help here.

**Comment by Gusted** [comment, marked as answer]
The dev tools aren't recommended for this, as the changes aren't permanent.

**Reply by OriLiMu** [reply]
ok, if I want to make the dark part more black, can I do that? For example the dark reader change #fff to #aaa currently, I want to make it to #666. Is this possible?

**Reply by Gusted** [reply]
That's possible:


    Click Dark Reader icon.
    Click Dev tools (in the bottom-right corner).
    Click Preview new design.


Custom background, text and selection colors (under Theme click See all options, Colors).
